From 028a0e9c399f2a2ac32f47bc5523f0ee7d692e08 Mon Sep 17 00:00:00 2001
From: Ernestas Kulik <ernestask@src.gnome.org>
Date: Sun, 8 Jan 2017 17:20:20 +0200
Subject: general: use own icon instead of system-file-manager

Since adwaita-icon-theme no longer ships the system-file-manager icon,
we must our own. 7a9eacc0afa53520eef12542c4d330bf66142200 was to include
this change, as part of Flatpak integration.

https://bugzilla.gnome.org/show_bug.cgi?id=777011
---
 data/org.gnome.Nautilus.desktop.in.in |    2 +-
 src/nautilus-application.c            |    2 +-
 src/nautilus-preferences-window.c     |    2 +-
 src/nautilus-window.c                 |    2 +-
 src/resources/ui/nautilus-window.ui   |    2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

--- a/data/org.gnome.Nautilus.desktop.in.in
+++ b/data/org.gnome.Nautilus.desktop.in.in
@@ -3,7 +3,7 @@ _Name=Files
 _Comment=Access and organize files
 _Keywords=folder;manager;explore;disk;filesystem;
 Exec=nautilus --new-window %U
-Icon=system-file-manager
+Icon=org.gnome.Nautilus
 Terminal=false
 Type=Application
 DBusActivatable=true
--- a/src/nautilus-application.c
+++ b/src/nautilus-application.c
@@ -1230,7 +1230,7 @@ nautilus_application_startup (GApplicati
 	 */
 	G_APPLICATION_CLASS (nautilus_application_parent_class)->startup (app);
 
-	gtk_window_set_default_icon_name ("system-file-manager");
+	gtk_window_set_default_icon_name ("org.gnome.Nautilus");
 
 	setup_theme_extensions ();
 
--- a/src/nautilus-preferences-window.c
+++ b/src/nautilus-preferences-window.c
@@ -529,7 +529,7 @@ static void nautilus_preferences_window_
   window = GTK_WIDGET(gtk_builder_get_object(builder, "preferences_window"));
   preferences_window = window;
 
-  gtk_window_set_icon_name(GTK_WINDOW(preferences_window), "system-file-manager");
+  gtk_window_set_icon_name(GTK_WINDOW(preferences_window), "org.gnome.Nautilus:");
 
   g_object_add_weak_pointer(G_OBJECT(window), (gpointer *)&preferences_window);
 
--- a/src/nautilus-window.c
+++ b/src/nautilus-window.c
@@ -2769,7 +2769,7 @@ nautilus_window_show_about_dialog (Nauti
 				 * box to give credit to the translator(s).
 				 */
 			      "translator-credits", _("translator-credits"),
-			      "logo-icon-name", "system-file-manager",
+			      "logo-icon-name", "org.gnome.Nautilus",
 			      NULL);
 }
 
--- a/src/resources/ui/nautilus-window.ui
+++ b/src/resources/ui/nautilus-window.ui
@@ -3,7 +3,7 @@
   <template class="NautilusWindow" parent="GtkApplicationWindow">
     <property name="show-menubar">False</property>
     <property name="title" translatable="yes">_Files</property>
-    <property name="icon-name">system-file-manager</property>
+    <property name="icon-name">org.gnome.Nautilus</property>
     <child type="titlebar">
       <object class="NautilusToolbar" id="toolbar">
         <!-- We override the title widget so the GtkHeaderBar stop managing the
